.github/workflows/rust.yml

```diff
@@ -27,6 +27,8 @@ jobs:
     - name: Run Clippy against main lib
       run: cargo clippy --workspace --exclude tests
     - name: Run tests
-      run: cargo test --verbose
+      run: cargo test --verbose -- --test-threads=1
```

Comment:
Needed `test-threads=1` to prevent races around the meetme service when tests are kicked off in parallel. In the future, we could just select an unallocated port per test.